sdk/storage/azure-storage-file-datalake/src/main/java/com/azure/storage/file/datalake/models/FileSystemProperties.java

```diff
@@ -152,4 +173,9 @@ public Boolean isEncryptionScopeOverridePrevented() {
         return encryptionScopeOverridePrevented;
     }
 
+    private FileSystemProperties setEncryptionScope(String encryptionScope) {
+        this.encryptionScope = encryptionScope;
+        return this;
+    }
```

Comment:
This method is not needed since we are directly updating the property in the accessor method.